datahub-web-react/src/app/ingest/source/builder/RecipeForm/common.tsx

```diff
@@ -357,3 +359,34 @@ export const SKIP_PERSONAL_FOLDERS: RecipeField = {
     fieldPath: 'source.config.skip_personal_folders',
     rules: null,
 };
+
+const NUM_CHARACTERS_TO_REMOVE_FROM_DATE = 5;
+
+const startTimeFieldPath = 'source.config.start_time';
+export const START_TIME: RecipeField = {
+    name: 'start_time',
+    label: 'Start Time',
+    tooltip:
+        'Earliest date of audit logs to process for usage, lineage etc. Default: Last full day in UTC or last time DataHub ingested usage (if stateful ingestion is enabled). Tip: Set this to an older date (e.g. 1 month ago) for your first ingestion run, and then uncheck it for future runs.',
+    placeholder: 'Select date and time',
+    type: FieldType.DATE,
+    fieldPath: startTimeFieldPath,
+    rules: null,
+    getValueFromRecipeOverride: (recipe: any) => {
+        const isoDateString = get(recipe, startTimeFieldPath);
+        if (isoDateString) {
+            return moment(isoDateString);
+        }
+        return isoDateString;
+    },
+    setValueOnRecipeOverride: (recipe: any, value?: Moment) => {
+        if (!value) {
+            return setFieldValueOnRecipe(recipe, null, startTimeFieldPath);
+        }
+        const isoDateString = value.toISOString();
+        const formattedDateString = isoDateString
```

Comment:
Can you see this being used elsewhere (and should it be in a utils file)?

Reply:
I certainly can - good call